policy/modules/admin/tomoyo-auditd.if

```diff
@@ -0,0 +1,27 @@
+## <summary>Audit log daemon for TOMOYO Linux MAC.</summary>
+
+########################################
+## <summary>
+##	All of the rules required to
+##	administrate tomoyo-auditd.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`tomoyo_auditd_admin',`
+	gen_require(`
+		type tomoyo_auditd_t;
+		type tomoyo_auditd_etc_t;
+		type tomoyo_auditd_log_t;
+	')
+
+	admin_pattern($1, { tomoyo_auditd_etc_t })
+
+	logging_search_logs($1)
+	admin_pattern($1, tomoyo_auditd_log_t)
```

Comment:
ironally enough you actualy use tomoyo_auditd_log_t here but you reference tomoyo_auditd_var_log_t in the fc spec

Reply:
Heh, fixed